lib/testing/src/lib/core/pages/data-table-component.page.ts

```diff
@@ -300,6 +300,16 @@ export class DataTableComponentPage {
         return row.click();
     }
 
+    doubleClickRowByContent(content) {
+        const row = this.getCellByContent(content);
+        BrowserVisibility.waitUntilElementIsVisible(row);
+        BrowserVisibility.waitUntilElementIsClickable(row);
+        row.click();
```

Comment:
can we just use BrowserActions.click(row) instead of 

```
BrowserVisibility.waitUntilElementIsVisible(row);
        BrowserVisibility.waitUntilElementIsClickable(row);
        row.click();
```